node/src/providers/ecdsa/triple.rs

```diff
@@ -19,6 +20,8 @@ use threshold_signatures::protocol::Participant;
 
 pub struct TripleStorage(DistributedAssetStorage<PairedTriple>);
 
+// IMPORTANT: if we add domain_ids as an idetifier to triples, ensure to also update the asset cleanup mechanism on startup (the other place where this constant is used).
+pub const TRIPLE_STORE_DOMAIN_ID: Option<DomainId> = None;
```

Comment:
It feels weird to have a const "None" value. Couldn't we just remove the option and argument to the `TripleStorage`?

Reply:
There already is no such option for the `TripleStorage` constructor: https://github.com/near/mpc/blob/9250afd3f6055d2e726908310c7a064f112a0e35/node/src/providers/ecdsa/triple.rs#L26